lib/generator.js

```diff
@@ -893,6 +893,9 @@
      *     Note that this option *cannot* be used with an inputRect/outputRect scaling. If inputRect/outputRect
      *     is set, this setting will be ignored and the pixels will not be cropped to document bounds.
      *     (Default: false)
+     * @param {?integer} settings.maxDimension: This is the maximal dimension of pixmap that can be returned
```

Comment:
The property on the `settings` object is a `number` not an `integer` (there is no integer type in JS). Also, the property is optional, not nullable, so it should be `number=`, not `?number`